src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -39,5 +39,23 @@ public static void EnableCollectFunction(this SymbolTable symbolTable)
         {
             symbolTable.AddFunction(new CollectFunction());
         }
+
+        /// <summary>
+        /// Enable a Patch() function which allows scripts to append table records.
+        /// </summary>
+        /// <param name="symbolTable"></param>
+        public static void EnablePatchRecordFunction(this SymbolTable symbolTable)
```

Comment:
>EnablePatchRecordFunction [](http://example.com/codeflow?start=27&length=25)

nit: we can probably include this w/ Collect.
And just generalize to: 
EnableMutationFunctions()

Clients will use these altogether.